src/components/navBar/navBar.js

```diff
@@ -414,6 +418,7 @@ function MdNavItem($$rAF) {
       'mdNavHref': '@?',
       'mdNavSref': '@?',
       'name': '@',
+      'disabled': '=?ngDisabled',
```

Comment:
I've pushed support for the disabled attribute, but the implementation does feel kinda hacky because I can't just get the disabled attribute from the element.
I've tried using $observe on the disabled attribute (and then using disabled: '@?' instead of nativeDisabled but I guess this version should be more performant.

I will add tests when the implementation is finished and reviewed.